### ISSUE TITLE
stable-0.4 bump version tag to 0.4.17.2

### ISSUE THREAD

**Issue by tverrbjelke** [minetest/minetest#7779]
##### Issue type
 - Bug report
 
##### Minetest version
0.4.17.1

##### OS / Hardware

Operating system: Debian 9
CPU: amd64 12 core

##### Summary
fresh install of 0.4.17.1 crashed on startup, where the fix was to not checkout 0.4.17.1 but instead checkout stable-0.4.

So something has been fixed, which should be worth a new tag. ?

The error already is described (and howto fix was mentioned) in [minetest_game issue 2234]( https://github.com/minetest/minetest_game/issues/2234)

Yesterday I asked this in the (minetest_game issue 2235](https://github.com/minetest/minetest_game/issues/2235) - and actually forgot to ask this HERE in minetest repo, too :-o

So the **minetest_game** branch  stable-0.4 already got the new tag 0.4.17.2, but it should be done here, too! ?

### steps to reproduce 
see my script [compile_minetest.sh](https://github.com/tverrbjelke/compile_minetest.git)

**Comment by paramat** [comment]
MTG versions 0.4.17.1, 0.4.17.2, 0.4.17 and branch stable-0.4 all point to the same commit, so those 4 versions are identical.
MTEngine 0.4.17.2 only exists as a special bugfix build for Android, i complained at the time it should not be called 0.4.17.2.

**Comment by paramat** [comment]
Checked:
'0.4.17.1' is at 10 June.
'stable-0.4' branch is at 28 June at the point of Android 0.4.17.2 release, it fixes the apparently Android-only bug detailed here #7439 
I think we should add the 0.4.17.2 tag here too. It's already causing confusion, as i warned :] I would do it but i would like to discuss with other core devs first the best approach.
> fresh install of 0.4.17.1 crashed on startup, where the fix was to not checkout 0.4.17.1 but instead checkout stable-0.4.
So something has been fixed, which should be worth a new tag. ?
The error already is described (and howto fix was mentioned) in minetest_game issue 2234

No, the error in MTG issue 2234 was simply caused by you using unmatched MTEngine and MTGame. Nothing was fixed.

**Comment by tverrbjelke** [comment]
I (my script) was compiling this minetest AND minetest Game for my desktop linux Debian stretch 9 (stable) from source. maybe you can have a look what the script is doing. 
And crashed with minetest 0.4.17.1 minetest and it did *not* crash with stable-0.4 when my script echecked out both  minetest and minetest-game and build the binareis. So **something** was fixed that also fixes my NON Android desktop!

And looking at the minetest-game, yes it is the same commit for stable-0.4 and 0.4.17.1 and 0.4.17.2.

But for minetest stable-0.4 holds some commits after 0.4.17.1 - and I would like to be able to checkout same version tag for minetest and minetest-game repositories. Currently there is not tag that works on m,y desktop, only the HEAD of branch stable-0.4 works. 

Maybe you have a look at the attached log of the mentioned other issue. There is the error - it has something to do with things from 0.5 getting activated in 0.4.17.1 ("firefly") and not in stable-0.4.
For my Desktop-Debian non Android system!

**Comment by paramat** [comment]
Thanks that helps explain.

**Comment by paramat** [comment]
So we need to add the 0.4.17.2 tag to this commit in branch 0.4-stable dd5f03731ad545f52da24825a2ee870f899f6adc

**Comment by tverrbjelke** [comment]
How fast you respond to that already "closed" ticket - I already feared that this issue went down the sink because no one reads closed issues. But, gladly you!

Some further success info for you!

I deployed from sources: 

 * minetest  branch 0.4-stable commit dd5f03731ad545f52da24825a2ee870f899f6adc 
 * together with minetest_game tag 0.4.17.2
 * added recent versions of some global mods: dreambuilder, vehicles, mobs_redo with "mobs_animal" "mobs_monster" "mobs_npc" "mob_horse", 3d_armor, rangedweapons
 * And then copied in the 3 yearsof age world (mapgen V6), re-activated the server, and 3 players did their stuff for a whole day (>5h) and they reported **"All  satisfied, no bug, no crash, all went nicely!"**
